### DISCUSSION TITLE
Prefered way to upgrade to 2.12.0

### DISCUSSION THREAD

**Discussion by luke-on-security** [monarc-project/MonarcAppFO#444] [Q&A] [answered]
with the latest changes in MONARC instalation method what is the "default"/preferred way to upgrade a MONARC instance (in my VirtualBOx VM)  ??

**Comment by cedricbonhomme** [comment, marked as answer]
As usual, [here](https://www.monarc.lu/documentation/technical-guide/#update-a-single-monarc-instance).

The only difference for "MONARC FO only users", is that now you can (optionally) install MONARC from a "bin" release. Which is proposed as a .tar.gz file, [here](https://github.com/monarc-project/MonarcAppFO/releases/latest). It's a more ["direct way"](https://www.monarc.lu/download/#source-code).

Let us know if you have any issue !
